doc/cloud-api.swagger

```diff
@@ -114,7 +114,7 @@ definitions:
       domain:
         type: "string"
         description: "Domain part of the conference.  Used if 'conference' is not provided.  Defaults to domain of the API endpoint.   Used to generate a 'conference' value (search by conference)"
- 
+
   ConferenceMapperDetails:
```

Comment:
I recall something about a calendar extension consuming the same api. It had support for both an array of phone numbers and a single number. To confirm, the change in this PR would make it so the single number version is no longer returned but the array of numbers version is returned?

Reply:
Yeah, anyway that will main use will be from the calendar-addon and the chrome extension will also be removed at some point (well Outlook is still there).